python/foglamp/services/core/api/notification.py

```diff
@@ -187,6 +187,13 @@
         is_enabled = "true" if ((type(enabled) is str and enabled.lower() in ['true']) or (
             (type(enabled) is bool and enabled is True))) else "false"
 
+        storage = connect.get_storage_async()
+        config_mgr = ConfigurationManager(storage)
+        curr_contig = await config_mgr.get_category_all_items(name)
+
+        if curr_contig is not None:
```

Comment:
curr_contig > category_exists

`if category_exists is not None:` can be simply `if category_exists:`